ksqldb-functional-tests/src/test/resources/query-validation-tests/avro.json

```diff
@@ -1183,6 +1183,118 @@
+          "valueFormat": "AVRO",
+          "valueSchemaId": 102,
+          "valueSchema": {"fields":[{"name":"eventID","type":"string"},{"name":"transaction","type":{"fields":[{"name":"num_shares","type":["null","int"]},{"name":"amount","type":["null","int"]},{"name":"customer","type":{"fields":[{"name":"first_name","type":"string"},{"name":"last_name","type":"string"},{"name":"email","type":"string"}],"name":"customer","type":"record"}}],"name":"transaction","type":"record"}}],"name":"Transaction","namespace":"com.acme.namespace","type":"record"}
+        }
+      ],
+      "inputs": [
+        {"topic": "transaction", "key": null, "value": {"id": "1", "transaction": {"txn_ts": "2020-11-18 02:31:43", "customer": {"first_name": "Jill", "last_name": "Smith", "id": 1234567, "email": "jsmith@foo.com" }, "company": {"name": "ACME Corp", "ticker": "ACMC", "id": "ACME837275222752952", "address": "Anytown USA, 333333"}}}},
+        {"topic": "transaction", "key": null, "value": {"id": "2", "transaction": {"txn_ts": "2020-11-18 02:35:43", "customer": { "first_name": "Art", "last_name": "Vandeley", "id": 8976612, "email": "avendleay@foo.com" }, "company": { "name": "Imports Corp", "ticker": "IMPC", "id": "IMPC88875222752952", "address": "Anytown USA, 333333"}}}},
+        {"topic": "transaction", "key": null, "value": {"id": "3", "transaction": {"txn_ts": "2020-11-18 02:36:43", "customer": { "first_name": "John", "last_name": "England", "id": 456321, "email": "je@foo.com" }, "company": { "name": "Hechinger", "ticker": "HECH", "id": "HECH8333785222752952", "address": "Anytown USA, 333333"}}}},
+        {"topic": "transaction", "key": null, "value": {"id": "4", "transaction": {"txn_ts": "2020-11-18 02:37:43", "customer": { "first_name": "Fred", "last_name": "Pym", "id": 333567, "email": "fjone@foo.com" }, "company": { "name": "PymTech", "ticker": "PYMT", "id": "PYME837275222714197419202020", "address": "Anytown USA, 333333"}}}}
+      ],
+      "outputs": [
+        {"topic": "model-transaction", "key": "1", "value": {"eventID": "1", "transaction": {"amount": null, "num_shares": null, "customer": {"last_name": "Smith", "first_name": "Jill", "email": "jsmith@foo.com"}}}},
+        {"topic": "model-transaction", "key": "2", "value": {"eventID": "2", "transaction": {"amount": null, "num_shares": null, "customer": {"last_name": "Vandeley", "first_name": "Art", "email": "avendleay@foo.com"}}}},
+        {"topic": "model-transaction", "key": "3", "value": {"eventID": "3", "transaction": {"amount": null, "num_shares": null, "customer": {"last_name": "England", "first_name": "John", "email": "je@foo.com"}}}},
+        {"topic": "model-transaction", "key": "4", "value": {"eventID": "4", "transaction": {"amount": null, "num_shares": null, "customer": {"last_name": "Pym", "first_name": "Fred", "email": "fjone@foo.com"}}}}
+      ]
+    },
+    {
+      "name": "VALUE_SCHEMA_ID with compound types and default value in the end of schema",
```

Comment:
@lihaosky added this test which tests https://github.com/confluentinc/ksql/pull/9575/files#diff-07daa80e9e013e949f189df69d12f918b084b5a1e821075b03a90a91321d569eR127

Reply:
Thanks. LGTM! Please check the build failure before merge

Reply:
Done